docs/pywbemcligeneraloptions.rst

```diff
@@ -92,8 +90,7 @@ information on each command-group arguments and options):
   can be used to execute pywbemcli commands without access to a real server.
   See chapter :ref:`Mock WBEM server support` for information on defining
   mock servers.
```

Comment:
For clarity and consistency, a statement about the mutual exclusiveness of the three options should be added here as well.

Reply:
DONE.